### ISSUE TITLE
Given iframe names in page are inconsistent

### ISSUE THREAD

**Issue by amerikan** [casperjs/casperjs#1121]
Not sure why, but when running the tests, seems casperjs sometimes fails to grab the iframe name even if it exists. It just randomly fails.

I ran my tests and logged the frame stack using:

``` js
this.page.childFramesName().forEach(function(n) {
    console.log(n);
});
```

Some runs I get this (fails to get name just shows as `<!--framePath //<!--frame9-->-->`):

``` shell
<!--framePath //<!--frame0-->-->
<!--framePath //<!--frame1-->-->
<!--framePath //<!--frame2-->-->
<!--framePath //<!--frame3-->-->
<!--framePath //<!--frame4-->-->
<!--framePath //<!--frame5-->-->
olark-loader
_olark_framesocket30988
<!--framePath //<!--frame9-->-->
```

Other runs it does get the frame name which is `upload_frame`:

``` shell
<!--framePath //<!--frame0-->-->
<!--framePath //<!--frame1-->-->
<!--framePath //<!--frame2-->-->
<!--framePath //<!--frame3-->-->
<!--framePath //<!--frame4-->-->
<!--framePath //<!--frame5-->-->
olark-loader
_olark_framesocket18180
upload_frame
```

**Using:**
casperjs 1.1.0-beta3
phantomjs 1.9.8
OSX 10.10.1

**Comment by kensoh** [comment]
Hi @amerikan not sure if this is still happening. I'm thinking could it be page elements/frames are still in process of loading and whether adding an explicit wait for some time before running this will help?

**Comment by kensoh** [comment]
Hi @amerikan not sure if this issue is resolved for you with latest CasperJS v1.1.4 and PhantomJS v2.1.1? Closing this for now but please reopen if it is still happening. Thanks for raising this!
